### PR TITLE
add some yaml config

### PR DESCRIPTION
**Pull request by phlrain**
### PR types
Function optimization

### PR changes
Others 

### Describe
add some yaml config

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_accuracy_op.py [hunk 1]
```diff
@@ -126,6 +128,18 @@ def test_api(self):
 
             self.assertEqual((result.numpy() == expect_value).all(), True)
 
+            # with _test_eager_guard():
```
Comment (phlrain) [inline python/paddle/fluid/tests/unittests/test_accuracy_op.py:131]: eager test，目前accuracy存在bug，待修复

**Review by zhiqiu — DISMISSED** [review]
LGTM for `atol` CI checking

**Review by JiabinYang — DISMISSED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/test_erf_op.py [hunk 2]
```diff
@@ -38,33 +39,33 @@ def _init_dtype(self):
...
-class TestErfLayer(unittest.TestCase):
-    def _test_case(self, place):
-        x = np.random.uniform(-1, 1, size=(11, 17)).astype(np.float64)
-        y_ref = erf(x)
-        with dg.guard(place) as g:
-            x_var = dg.to_variable(x)
-            y_var = fluid.layers.erf(x_var)
-            y_test = y_var.numpy()
-        self.assertTrue(np.allclose(y_ref, y_test))
+# class TestErfLayer(unittest.TestCase):
```
Comment (chenwhql) [inline python/paddle/fluid/tests/unittests/test_erf_op.py:48]: 注释掉的单测需要打开？

File: python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py [hunk 3]
```diff
@@ -30,6 +31,7 @@ class TestSigmoidCrossEntropyWithLogitsOp1(OpTest):
 
     def setUp(self):
         self.op_type = "sigmoid_cross_entropy_with_logits"
+        self.python_api = paddle.fluid.layers.sigmoid_cross_entropy_with_logits
```
Comment (chenwhql) [inline python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py:34]: 是不是应该尽可能用2.0 API，这里是binary_cross_entropy_with_logits

File: python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py [hunk 4]
```diff
@@ -61,6 +63,7 @@ class TestSigmoidCrossEntropyWithLogitsOp2(OpTest):
...
```
Comment (chenwhql) [inline python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py:66]: 下同

File: python/paddle/metric/metrics.py [hunk 5]
```diff
@@ -798,8 +798,12 @@ def accuracy(input, label, k=1, correct=None, total=None, name=None):
             total = _varbase_creator(dtype="int32")
 
         topk_out, topk_indices = paddle.topk(input, k=k)
+        # if _in_eager_mode():
```
Comment (chenwhql) [inline python/paddle/metric/metrics.py:801]: 这里是无用注释吗？还是暂时跑不通

File: python/paddle/tensor/linalg.py [hunk 6]
```diff
@@ -1447,7 +1451,10 @@ def bincount(x, weights=None, minlength=0, name=None):
     if x.dtype not in [paddle.int32, paddle.int64]:
         raise TypeError("Elements in Input(x) should all be integers")
 
-    if paddle.in_dynamic_mode():
+    # if in_dygraph_mode():
```
Comment (chenwhql) [inline python/paddle/tensor/linalg.py:1454]: 同上，如果是暂时跑不通需要留注释，同时加TODO说明下

File: python/paddle/fluid/tests/unittests/test_where_index.py [hunk 7]
```diff
@@ -21,15 +21,17 @@
 from paddle.fluid.op import Operator
 import paddle.fluid as fluid
 from paddle.fluid import Program, program_guard
+import paddle
 
 
 class TestWhereIndexOp(OpTest):
     def setUp(self):
         self.op_type = "where_index"
+        self.python_api = paddle.fluid.layers.where
```
Comment (chenwhql) [inline python/paddle/fluid/tests/unittests/test_where_index.py:30]: 2.0 API paddle.where

File: python/paddle/tensor/linalg.py (hunk 6 above)
Comment (phlrain) [reply python/paddle/tensor/linalg.py:1454]: 这个不是跑不通，是yaml被infrt 拦了 所以注释了
